src/auto-evo/RunResults.cs

```diff
@@ -811,6 +826,144 @@ void OutputPopulationForPatch(Species species, Patch patch, long population)
+                            }
+                        }
+                    }
+
+                    if (globalPopulation <= 0)
+                    {
+                        // TODO: see https://github.com/Revolutionary-Games/Thrive/issues/2958
+                        LogEventGloballyAndLocally(world, patch, new LocalizedString(
+                                "TIMELINE_SPECIES_EXTINCT", species.FormattedName),
+                            species.PlayerSpecies, "extinction.png");
+                    }
+                    else
+                    {
+                        if (hasPopulationBeforeMigration)
+                        {
+                            if (finalPatchPopulation > previousPatchPopulation)
+                            {
+                                patch.LogEvent(new LocalizedString("TIMELINE_SPECIES_POPULATION_INCREASE",
+                                        species.FormattedName, ((double)finalPatchPopulation).FormatNumber()),
+                                    species.PlayerSpecies, "popUp.png");
+                            }
+                            else
+                            {
+                                patch.LogEvent(new LocalizedString("TIMELINE_SPECIES_POPULATION_DECREASE",
+                                        species.FormattedName, ((double)finalPatchPopulation).FormatNumber()),
+                                    species.PlayerSpecies, "popDown.png");
+                            }
+                        }
+                        else
+                        {
+                            patch.LogEvent(new LocalizedString(
+                                    "TIMELINE_SPECIES_EXTINCT_LOCAL", species.FormattedName),
+                                species.PlayerSpecies, "extinctionLocal.png");
+                        }
+
+                        if (globalPopulation > previousGlobalPopulation)
+                        {
+                            world.LogEvent(new LocalizedString("TIMELINE_SPECIES_POPULATION_INCREASE",
+                                    species.FormattedName, ((double)globalPopulation).FormatNumber()),
+                                species.PlayerSpecies, "popUp.png");
+                        }
+                        else
+                        {
+                            world.LogEvent(new LocalizedString("TIMELINE_SPECIES_POPULATION_DECREASE",
+                                    species.FormattedName, ((double)globalPopulation).FormatNumber()),
+                                species.PlayerSpecies, "popDown.png");
+                        }
+                    }
+                }
+
+                foreach (var migration in GetMigrationsTo(patch))
+                {
+                    // Log to destination patch
+                    patch.LogEvent(new LocalizedString("TIMELINE_SPECIES_MIGRATED_FROM", migration.Key.FormattedName,
+                            TranslationServer.Translate(migration.Value.From.Name)),
```

Comment:
This should also use a nested LocalizedString so that this reacts to language change properly.